docs-website/docusaurus.config.js

```diff
@@ -373,6 +373,12 @@ module.exports = {
           }
           return undefined; // Return a falsy value: no redirect created
         },
+        redirects: [
+          {
+            from: '/docs/managed-datahub/operator-guide/setting-up-remote-ingestion-executor',
+            to: '/docs/managed-datahub/remote-executor/about',
```

Comment:
once we can confirm this redirect works, we'll remove `/docs/managed-datahub/operator-guide/setting-up-remote-ingestion-executor`

Reply:
@yoonhyejin would love your help here - I don't think we have an established pattern for deprecating docs & configuring a URL redirect & it doesn't look like this is working; curious if you have ideas!

Reply:
@yoonhyejin pls disregard - going to workaround the existing file structure to retain the legacy path